app/Services/AuthServiceProvider.js

```diff
@@ -82,6 +83,25 @@ function validateLogin(req, res) {
   return validateAuth(req, res);
 }
 
+function validatePassword(password) {
+  const passwordLength = bytesLength(password);
+
+  const length = passwordLength >= 8 && passwordLength <= 72;
+  const hasUpperCase = /[A-Z]/.test(password);
+  const hasLowerCase = /[a-z]/.test(password);
+  const hasNumbers = /\d/.test(password);
+  const hasNonAlphas = /\W/.test(password);
+
+  // eslint-disable-next-line no-nested-ternary
+  const passwordComplexity = length
+    ? length + hasUpperCase + hasLowerCase + hasNumbers + hasNonAlphas
+    : passwordLength !== 0
+    ? 1
+    : 0;
```

Comment:
It wasn't the best way from me of doing this. Can you please try to write that with normal if statements? I'll also change that in my pr as well.